src/nvim/tui/tui.c

```diff
@@ -802,7 +802,16 @@ static void reset_scroll_region(UI *ui)
 static void tui_grid_resize(UI *ui, Integer g, Integer width, Integer height)
 {
   TUIData *data = ui->data;
-  ugrid_resize(&data->grid, (int)width, (int)height);
+  UGrid *grid = &data->grid;
+  ugrid_resize(data, (int)width, (int)height);
```

Comment:
why pass `data` instead of `grid` to `ugrid_resize` ?

Reply:
no reason :P